package.json

```
@@ -11,48 +11,38 @@
-    "report": "npm-run-all -s --continue-on-error clean:report report:**",
-    "report:build": "node scripts/report-build-info.js",
-    "report-disabled:coverage:flow": "flow-coverage-report --threshold 45",
-    "report:coverage:jest": "npm run test:unit -- --coverage",
-    "report:lighthouse": "lhci autorun",
-    "coveralls": "cat reports/jest-coverage/lcov.info | coveralls",
-    "dev": "webpack serve --mode=development",
-    "flow-typed:install": "npx flow-typed install",
-    "lint:fix": "npm run test:lint -- --fix",
+    "build-storybook": "echo 'STORYBOOK temporarily disabled'",
+    "bundle": "webpack --progress",
     "clean": "rimraf dist",
     "clean:report": "rimraf reports && rimraf .lighthouseci && mkdirp reports",
-    "bundle": "webpack --progress",
+    "create-sitemap": "node scripts/create-sitemap.js",
     "predeploy": "npm-run-all test clean",
-    "COMMENTED:postdeploy": "npm run test:functional",
     "deploy": "npm run bundle",
-    "inject-first-content": "node scripts/inject-first-content.js",
     "deploy:dev": "npm run deploy -- -- --mode=development",
     "deploy:staging": "npm run deploy -- -- --env staging",
     "deploy:production": "npm run deploy",
+    "dev": "webpack serve --mode=development",
+    "flow-typed:install": "npx flow-typed install",
+    "lint:fix": "npm run test:lint -- --fix",
     "release": "npm run bundle",
     "release:dev": "npm run release -- -- --mode=development && npm run build-storybook",
     "release:staging": "npm run release -- -- --env staging && npm run build-storybook",
     "release:production": "npm run release",
-    "create-sitemap": "node scripts/create-sitemap.js",
+    "report": "npm-run-all -s --continue-on-error clean:report report:**",
+    "report:build": "node scripts/report-build-info.js",
+    "report:coverage:jest": "npm run test:unit -- --coverage",
+    "report:lighthouse": "lhci autorun",
     "serve": "node scripts/server.js",
     "start": "npm run dev",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "echo 'TMP DISABLED: build-storybook --output-dir dist/storybook'",
-    "postinstall": "husky install; patch-package",
-    "prepublishOnly": "pinst --disable",
-    "postpublish": "pinst --enable"
+    "test": "npm-run-all test:flow test:lint test:lint:ts test:unit",
+    "test:flow": "flow focus-check --merge-timeout=1000 .",
+    "test:lint": "eslint --color --ignore-path .gitignore --ignore-pattern \"flow-typed\" --ignore-pattern \"lighthouserc.js\" --ignore-pattern \"snippets\" \"./**/*.js\"",
+    "test:lint:ts": "eslint --color --config .eslint4ts.js --no-eslintrc ./src/**/*.ts ./src/**/*.tsx",
+    "test:unit": "jest --maxWorkers=2 src",
+    "version": "node --version && npm --version",
+    "postinstall": "husky install; patch-package"
```

```suggestion
    "postinstall": "husky install && patch-package"
```

Or maybe just remove it? Is it used anywhere?

---

postinstall is one of those magic commands, it runs as its name says after  `npm install`, so its useful to configure stuff in the project at that point, in this case we are using it for 2 things:
* install husky, to have things like lint and prettier run when committing or pushing.
* running `patch-package` which we use to patch the `EBI-Framework 1.4` to edit a bit of its code and avoid some annoying warnings at build.

---

why is it better `&&` over `;`?

---

Based on [this explanation](https://superuser.com/questions/619016/what-is-the-difference-between-command-command-and-command-command/619019#619019) I think it is better to use `;` in this case, because even if `husky` fails installing, the `patch-package` will run.

---

> postinstall is one of those magic commands, it runs as its name says after `npm install`, so its useful to configure stuff in the project at that point, in this case we are using it for 2 things:
> 
>     * install husky, to have things like lint and prettier run when committing or pushing.
> 
>     * running `patch-package` which we use to patch the `EBI-Framework 1.4` to edit a bit of its code and avoid some annoying warnings at build.

My bad, I didn't realize it was a built-in feature.

> Based on [this explanation](https://superuser.com/questions/619016/what-is-the-difference-between-command-command-and-command-command/619019#619019) I think it is better to use `;` in this case, because even if `husky` fails installing, the `patch-package` will run.

Fair enough.